data_collection/gazette/spiders/pr/pr_juranda.py

```diff
@@ -0,0 +1,10 @@
+from datetime import date
+
+from gazette.spiders.base.atende_v2 import BaseAtendeV2Spider
+
+
+class PrJurandaSpider(BaseAtendeV2Spider):
+    TERRITORY_ID = "4112959"
+    name = "pr_juranda"
+    city_subdomain = "juranda"
+    start_date = date(2012, 2, 3)
```

Comment:
```suggestion
    start_date = date(2021, 3, 24)
```

Pelo que naveguei no site (https://juranda.atende.net/diariooficial/edicao/situacaoEdicao/undefined/pagina/29) a primeira edição é de 24/março/2021. Pelo arquivo de coleta completa anexado à PR, também -- é a data mais antiga da tabela.